osprey/processes/wps_convolution.py

```diff
@@ -29,6 +30,56 @@ def log_handler(process, response, message, process_step=None, level="INFO"):
 
 class Convolution(Process):
     def __init__(self):
+        self.config_dict = {
```

Comment:
This is _fine_, but generally we don't want to name vars with its type in the name.